--- seafile-5.1.2-server/./lib/Makefile.am~	2016-05-10 15:25:35.000000000 +0300
+++ seafile-5.1.2-server/./lib/Makefile.am	2016-06-07 23:22:19.846875742 +0300
@@ -64,7 +64,7 @@
 	@rm -f rpc_table.tmp
 	@touch rpc_table.tmp
 	@echo "[libsearpc]: generating rpc header files"
-	@PYTHON@ `which searpc-codegen.py` ${top_srcdir}/lib/rpc_table.py
+	@PYTHON@ /usr/bin/searpc-codegen ${top_srcdir}/lib/rpc_table.py
 	@echo "[libsearpc]: done"
 	@mv -f rpc_table.tmp $@
 
